ddtrace.gemspec

```diff
@@ -53,13 +53,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pimpmychangelog', '>= 0.1.2'
   spec.add_development_dependency 'appraisal', '~> 2.2'
   spec.add_development_dependency 'yard', '~> 0.9'
-  spec.add_development_dependency 'webmock', '~> 2.0'
+  spec.add_development_dependency 'webmock', '>= 3.10.0'
   spec.add_development_dependency 'builder'
-  if RUBY_PLATFORM != 'java'
-    spec.add_development_dependency 'sqlite3', '~> 1.3.6'
-  else
-    spec.add_development_dependency 'jdbc-sqlite3', '~> 3'
-  end
```

Comment:
I did the overdue work of moving `sqlite3` to our Appraisal file: we cannot run all our test scenarios with version `~> 1.3.6` anymore. The version of `sqlite3` is now correctly handled in each Appraisal scenario.